audformat/core/database.py

```diff
@@ -583,6 +589,10 @@ def job(obj_id, obj):
                 task_description='Save tables',
             )
 
+            # Check attachments exist
+            for attachment_id in list(self.attachments):
+                self.attachments[attachment_id]._check_path(root)
```

Comment:
Is this is enough for folders or do we also need to check there are no symlinks within a folder? But probably it's sufficient to check this in `audb.publish()` as a user could anyway add files to a folder afterwards.

Reply:
Yes, I would indeed check only really necessary cases inside `audformat` and would do the rest in `audb`.